collect/cnrfc/cnrfc.py

```diff
@@ -850,6 +850,47 @@ def get_forecast_csvdata(url):
     return _get_forecast_csv(url)
 
 
+def get_rating_curve(cnrfc_id):
+    """
+    returns paired flow and stage data parsed from the text of CNRFC rating curve JavaScript files
+    
+    Arguments:
+        cnrfc_id (str): forecast point (such as FOLC1)
+    Returns:
+        result (dict): query result dictionary with 'data' and 'info' keys
+    """
+    # retrieve data from URL
+    url = f'https://www.cnrfc.noaa.gov/data/ratings/{cnrfc_id}_rating.js'
+    response = requests.get(url)
+
+    # check if data exists
+    if response.status_code == 200:
+        raw_data = response.text.splitlines()[2:]
```

Comment:
Since you are using `startswith` below, I don't think you need to explicitly specify the `[2:]`.

Reply:
Resolved in [1a296a7](https://github.com/MBKEngineers/collect/pull/68/commits/1a296a7d646d7a758fced757353de4751efb9964)